source/_integrations/modbus.markdown

```diff
@@ -38,9 +38,7 @@ The modbus integration provides a number of parameters to help communicate with
 
 {% configuration %}
 close_comm_on_error:
```

Comment:
If something is deprecated, we remove it from the documentation (with one exception: if the whole integration is deprecated, we place a warning box).

As the latter is not the case here, please remove this one.

../Frenck

Reply:
ok will do tomorrow.

Reply:
Hehe you have an odd tomorrow 😉 

Thanks for updating! ❤️ 

../Frenck